crates/pathfinder/src/rpc/v02.rs

```diff
@@ -151,6 +151,11 @@ where
 // Registers all methods for the v0.2 API
 pub fn register_all_methods(module: &mut jsonrpsee::RpcModule<RpcContext>) -> anyhow::Result<()> {
     register_method_with_no_input(module, "starknet_chainId", method::chain_id::chain_id)?;
+    register_method(
+        module,
+        "starknet_getClassHashAt",
+        method::get_class_hash_at,
```

Comment:
This is likely the reason why build fails: `method::get_class_hash_at::get_class_hash_at` should do.